### ISSUE TITLE
Nitro adds trailing slash at the end of devProxy target

### ISSUE THREAD

**Issue by wokalek** [nitrojs/nitro#2659]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v20.15.0
- Nuxt Version:     3.12.4
- CLI Version:      3.12.0
- Nitro Version:    2.9.7
- Package Manager:  pnpm@9.4.0
- Builder:          -
- User Config:      compatibilityDate, telemetry, devtools, srcDir, app, extends, modules, vite, nitro, appConfig, runtimeConfig, css, apollo, tailwindcss, postcss, colorMode, device, mdc, image, pwa, robots, sitemap, site
- Runtime Modules:  @nuxtjs/apollo@5.0.0-alpha.14, @nuxtjs/tailwindcss@6.12.1, @nuxtjs/robots@4.0.2, @nuxtjs/sitemap@5.3.5, @nuxtjs/color-mode@3.4.2, @nuxtjs/fontaine@0.4.3, @nuxtjs/device@3.1.1, @nuxtjs/mdc@0.8.3, @nuxt/image@1.7.0, @pinia/nuxt@0.5.3, @vueuse/nuxt@10.11.1, @vite-pwa/nuxt@0.9.1, nuxt-typed-router@3.6.5, nuxt-svgo@4.0.3, nuxt-schema-org@3.3.9, nuxt-lodash@2.5.3
- Build Modules:    -
------------------------------

### Reproduction

Set the settings from my settings described below and make sure that the proxy request is sent to the wrong url from nitro side.

### Describe the bug

nuxt config:

```js
nitro: {
  devProxy: {
    '/graphql': 'http://localhost:8000/graphql',
  },
},
```

The request from the browser goes to `http://localhost:3000/graphql`, but in the response I see (from Django) that the request came to `http://localhost:3000/graphql/` and because my route on Django is set specifically to `/graphql`, it cannot be resolved.

I can add `\/?` to the expression on the backend side, but this is `http://localhost:8000/graphql/` not what I wrote in the devProxy settings. This is a different url.

How to solve the problem? Or is it a bug?

Request in headers tab: 
![image](https://github.com/user-attachments/assets/9812dcaf-3a6f-4d9e-930b-d31d716556ab)

Request preview (Django response): 
![image](https://github.com/user-attachments/assets/192ffa16-4bd3-46e2-a827-7edd6334743e)
Django logs:
![image](https://github.com/user-attachments/assets/a42e4da7-b7e7-4e46-b3f1-d4570c178372)

**Comment by wokalek** [comment]
After deeping down inside nitro and after httpxy, the following setting helped:

```js
  '/graphql': { target: 'http://localhost:8000/graphql', prependPath: false },
```

**Comment by wokalek** [comment]
No, now it proxies requests to just `http://localhost:8000`. That's not what I need.

**Comment by bucherfa** [comment]
my solution to this is the following:

```js
  '/graphql': { target: 'http://localhost:8000/graphql', ignorePath: true },
```

The documentation is a little confusing, since nitro uses [httpxy](https://github.com/unjs/httpxy) which refers for documentation to [node-http-proxy](https://github.com/http-party/node-http-proxy?tab=readme-ov-file#options), where you need to look for the option `ignorePath`.

**Comment by wokalek** [comment]
@bucherfa It's been a while since the issue, but I still have the previous configuration.

I think I tried to use ignorePath, but I'll take another look. Thanks!
